src/bp/sdk/botpress.d.ts

```diff
@@ -1376,11 +1377,34 @@ declare module 'botpress/sdk' {
     timeoutNode?: string
     flow?: string
     prompt?: PromptNode
+    subflow?: SubWorkflowNode
     isNew?: boolean
     /** Used internally by the flow editor */
     readonly lastModified?: Date
   } & NodeActions
 
+  export interface SubWorkflowNode {
+    in: SubWorkflowInputs
+    out: SubWorfklowOutputs
+  }
+
+  export interface SubWorkflowInput {
+    /** Take the value of a variable present in the caller flow */
+    passVariable: string
+    /** Hardcode a value to be passed to the sub-workflow */
+    passValue: string
+  }
```

Comment:
Not sure about the naming of those variables, "pass" sounds weird. What do you think of this? 

```js
export interface SubWorkflowInput {
    source: 'variable' | 'hardcoded'
    value: any
  }
```